src/app.js

```diff
@@ -1 +1,33 @@
 'use strict';
+
+const readline = require('readline');
+const { randomNumber } = require('./getRandomNumber');
+const { bull } = require('./bull');
+const { cow } = require('./cow');
+
+// eslint-disable-next-line no-console
+const log = console.log;
```

Comment:
Delete it. I think this code was needed for debugging, but do not forget to delete it before do the pull request - here and in your future test tasks.